packages/framework/aqueduct/src/data-objects/pureDataObject.ts

```diff
@@ -190,18 +187,6 @@ export abstract class PureDataObject<I extends DataObjectTypes = DataObjectTypes
 		}
 	}
 
-	/**
-	 * Gets the service at a given id.
-	 * @param id - service id
-	 * @deprecated See https://github.com/microsoft/FluidFramework/pull/14430 for context
-	 */
-	protected async getService<T extends FluidObject>(id: string): Promise<T> {
```

Comment:
It's interesting that this change to a protected member doesn't trigger typetest failures - we would have just marked it as broken in this PR anyway, but seems like something that should have triggered. @Abe27342 @anthony-murphy FYI in case this gap is unknown.